monero-outreach-docs/translations/es/storyboard/refuge_es.md

```diff
@@ -0,0 +1,94 @@
+
+*Pero, ¿que sucedería si los activos y el dinero del individuo pudieran mantenerse en formato digital?* 
+
+Si no hay un activo tangible a plena vista para un agente de policía o un funcionario, la probabilidad de registro y su posible incautación disminuyen significativamente. 
+
+Ten en cuenta a Monero y las criptomonedas que preservan la privacidad. Mientras que la mayoría de las criptomonedas de persona a persona (p2p) están en el camino correcto hacia la autonomía de la codicia y la aplicación de las leyes de decomiso, aún no son inmunes. Han sido monitoreadas y rastreadas en cierto grado por [empresas privadas y agencias gubernamentales a través de los UTXOs (salida de transacción no gastada)](https://bitcoinist.com/yes-your-bitcoin-transactions-can-be-tracked-and-here-are-the-companies-that-are-doing-it/). 
+ 
+Sin embrago, Monero a través del [protocolo CryptoNote](https://cryptonote.org/) opera con diferencias muy marcadas (en comparación con Bitcoin) en la manera de ocultar información en la cadena de bloques. Mediante firmas de círculo se ocultan los inputs de una transacción entre otros múltiples posibles inputs, incrementando la dificultad de vínculos con transacciones seguidas. Mientras, [Malte Möser de la universidad de Princeton destacó que el 62% las transacciones](https://www.technologyreview.com/s/610807/sitting-with-the-cyber-sleuths-who-track-cryptocurrency-criminals/) de Monero fueron vulnerables al análisis de rastreo (abril del 2018), no fue por deficiencias en el protocolo sino por errores humanos y prácticas de privacidad descuidadas. 
+
+Vía firmas de círculo en el protocolo CryptoNote, llegado el caso que [oficiales y funcionarios incautaran un computador](https://www.heritage.org/economic-and-property-rights/commentary/civil-asset-forfeiture-has-created-serious-problems), eventualmente fallarían en el intento de descifrar el origen de los fondos, ya que todos los del grupo habrán sido posibles firmantes. La fungibilidad de Monero remueve de manera tajante, cualquier "[identificador exclusivo que pudiera atar las transacciones de una moneda de XMR en el tiempo.](https://www.monerooutreach.org/preserving-fundamental-rights.php)" 
+ 
+La subversión de las autoridades federales a cooperar en la promoción de los acuerdos de intercambio equitativo es probable que disminuya, cuando haya cero posibilidades (eliminando la manipulación humana) de que funcionarios recuperen los XMR propios de los individuos. 
+
+### **Castigos arbitrarios y/o desproporcionados** 
+
+Al resultado de la disminución del comportamiento codicioso de la policía y funcionarios del gobierno, le seguirán los castigos desproporcionados (muchas veces arbitrarios). Con menos activos que reclamar y señalados de participar en actividades ilícitas, menor el potencial para gravar y la ejecución de dichos activos. 

```

Comment:
A little suggestion, it's not important:

+_**Como**_ resultado de la disminución del comportamiento codicioso de la policía y funcionarios del gobierno, ~~le~~ seguirán los castigos desproporcionados...

Reply:
done